### ISSUE TITLE
 qiime quality-filter q-score (version 2018.4)

### ISSUE THREAD

**Issue by amitpande74** [galaxyproject/tools-iuc#2109]
Kindly install the wrappers at 
https://toolshed.g2.bx.psu.edu/repository/view_repository?changeset_revision=51b9b6b57732&id=a43ea4908816827c

**Comment by peterjc** [comment]
I don't think this issue tracker is the right place for your question - Who and where are you asking these to be installed?

**Comment by bgruening** [comment]
This issue was supposed to be "we need to update our Qiime wrappers" I guess.
